browser/js/popup.js

```diff
@@ -33,7 +33,12 @@ function routePopup(el, location) {
     // This query will only enter if the appropriate URL structure has already
     // been matched, so we can make some assumptions about the structure of
     // the URL.
-    if (location.hostname.endsWith('reddit.com')) {
+    if (location.hostname.startsWith('keybase.')) {
+        // For keybase.io and keybase.pub
+        const username = location.pathname.split('/')[1];
+        return renderPopup(el, username, 'keybase');
```

Comment:
did we talk about supporting the custom subdomains in keybase.pub? like `https://chris.keybase.pub/`